exchange/utils_test.go

```diff
@@ -868,6 +868,90 @@ func TestCleanOpenRTBRequestsSChain(t *testing.T) {
 	}
 }
 
+func TestCleanOpenRTBRequestsBidderParams(t *testing.T) {
+	testCases := []struct {
+		description string
+		inExt       json.RawMessage
+		expectedExt map[string]json.RawMessage
+		hasError    bool
+	}{
+		{
+			description: "Bidder params for single partner",
```

Comment:
These descriptions should be unique to each test, so you can identify the failing test if a test fails. Please modify the descriptions so the differences between the two tests are clear.

Reply:
Fixed this error.